srcpkgs/mupdf/template

```diff
@@ -1,7 +1,8 @@
 # Template file for 'mupdf'
+# Static library, revbump all dependants on mupdf updates
```

Comment:
We could do what Alpine does maybe: https://git.alpinelinux.org/aports/tree/community/mupdf/APKBUILD#n109

But not sure if it's worth it.